README.md

```diff
@@ -60,13 +60,32 @@ Currently supports importing Google Docs into a StreamField.
    from wagtail_content_import.mappers.streamfield import StreamFieldMapper
    
     class MyMapper(StreamFieldMapper):
-        html = RichTextConverter('my_RichTextBlock_name')
-        image = ImageConverter('my_ImageBlock_name')
-        heading = TextConverter('my_CharBlock_or_TextBlock_name')
-        table = TableConverter('my_TableBlock_name')    
+        html = RichTextConverter('my_heading_block')
+        image = ImageConverter('my_image_block')
+        heading = TextConverter('my_heading_block')
+        table = TableConverter('my_table_block')    
     ```
+   
+   This would map to an example StreamField defined as:
+   
+   ```python
+   from wagtail.images.blocks import ImageChooserBlock
+   from wagtail.core.blocks import CharBlock, RichTextBlock, StreamBlock
+   from wagtail.contrib.table_block.blocks import TableBlock
+   
+   class BaseStreamBlock(StreamBlock):
+    """
+    Define the custom blocks that `StreamField` will utilize
+    """
+    my_heading_block = CharBlock()
+    my_paragraph_block = RichTextBlock()
+    my_image_block = ImageChooserBlock()
+    my_table_block = TableBlock()
```

Comment:
missing indent

Reply:
Thanks! Oops, don't know why that didn't copy across - weird. Fixed now.